src/Commands/config/ConfigCommands.php

```diff
@@ -175,6 +176,85 @@ public function delete($config_name, $options = []) {
   }
 
   /**
+   * Display status of configuration (differences between stored and active configuration).
```

Comment:
The word "stored" is not too intuitive here. I think we might want to explain the default case and say "differences between the filesystem configuration and database configuration". Dunno.